src/views/catalog/CreateFromInstanceTypes/components/VMDetailsSection/components/DynamicSSHKeyInjectionIntanceType.tsx

```diff
@@ -11,6 +12,9 @@ import { PopoverPosition } from '@patternfly/react-core';
 const DynamicSSHKeyInjectionIntanceType = () => {
   const { instanceTypeVMState, setInstanceTypeVMState } = useInstanceTypeVMStore();
   const { sshSecretCredentials } = instanceTypeVMState;
+  const isDynamicSSHInjectionLabelExist =
```

Comment:
`isDynamicSSHInjectionLabelExist` feels to me kinda long and kinda weird. In English, we never ask the way "is something exist?", but "does something exist?" So having verbs "is" and "exist" in one name of the variable can be confusing. We could also ask if something exists like "is something present?" or ".. available"?

So I'd better suggest to name the variable for example `isDynamicSSHInjectionLabelPresent` (even if it is even a bit longer, but makes more sense in English and makes the code clearly understandable), or maybe something like `dynamicSSHKeyLabelExist` or `hasDynamicSSHLabel` (which would say the information that some related resource _has_ the appropriate related label or not). Up to you :) 

Note that this is not super important however I'd recommend to improve the name of this variable a little bit :)

Reply:
Ok Hilda 👍